com.unity.render-pipelines.universal/Shaders/Terrain/TerrainLit.shader

```diff
@@ -42,7 +42,7 @@ Shader "Universal Render Pipeline/Terrain/Lit"
 
 	HLSLINCLUDE
 
-	#pragma multi_compile_fragment __ _ALPHATEST_ON
```

Comment:
was this an specific issue to terrain or to any ALPHATEST_ON keyword?

Reply:
This is a specific issue to terrain shader because some terrain vertex shaders are using ALPHATEST_ON. eg: https://github.com/Unity-Technologies/Graphics/blob/63442bf01659e97ceccd447e290bd17a8538be16/com.unity.render-pipelines.universal/Shaders/Terrain/TerrainLitPasses.hlsl#L480

multi_compile only fragment shader in this case will cause logic to break.